src/ckb/schema/gen.sh

```diff
@@ -12,4 +12,9 @@ for file in "${files[@]}"; do
     f="$schema_dir/$file"
     output_file="$gen_dir/$(basename "${f%%.mol}.rs")"
     "$MOLC" --language rust --schema-file "$f" | rustfmt > "$output_file"
+
+    ## ignore them in clippy
+    DIRECTIVE="#![allow(clippy::all)]"
+    echo -e "$DIRECTIVE\n$(cat "$output_file")" > "$output_file"
```

Comment:
Reading and writing the same file in a single shell pipeline may cause problems like incomplete read. It is normally recommended to use sponge https://man.archlinux.org/man/sponge.1 . But here we can just replace the cat command with the above molc command.

Reply:
`sponge` is not a default tool on macos, fixed it with `mktemp`.

Reply:
please add `#[allow(clippy::all)]` to the gen/mod.rs thus we can keep the generated file unchanged.
```
#[allow(clippy::all)]
pub mod cfn;
```